Source/Other/Extensions/NumberFormatterExtension.swift

```diff
@@ -9,11 +9,16 @@
 import Foundation
 
 extension NumberFormatter {
-
 	@objc public static var decimalStyleFormatter: NumberFormatter = {
-			
 		let formatter = NumberFormatter()
 		formatter.numberStyle = .decimal
 		return formatter
-	}()
+	}
+
+	@objc static func withFormat(_ format: String) -> NumberFormatter {
```

Comment:
can we make this init instead of `withFormat(_ format:) func`? Swift doesn't really use `with` as it's automated, so `@objc init(format: String)` is for objective C translated as `initWithFormat:`

Reply:
Or just somehow rename it so it's clear it creates new formatter into memory...